### DISCUSSION TITLE
Export Operators to JSON-File

### DISCUSSION THREAD

**Discussion by schwamar** [Traewelling/traewelling#2029] [Ideas / Feature Requests]
*Problem:*
In the JSON-File an info about the carriers/operators is missing.

*Possible Solution:*
Include the operator-Object into the json-export file at the section data/status/train or at data/trip.

*Possible Alternatives*
Maybe it could also be included in the PDF-Export.

*Additional context*
In the year's review (many thanks to this very nice review!:) a counter for operators is included and also the most used operator is listed. So the data should be somewhere available in the database. ;)

**Comment by MrKrisKrisu** [comment]
Do you mean the export at https://traewelling.de/export?
There I can find the operator attributes.

```json
"operator":{"identifier":"albtal-verkehrs-gesellschaft-mbh","name":"Albtal-Verkehrs-Gesellschaft mbH"}
```
